### PR TITLE
Add ros2_control needed elements

### PR DESCRIPTION
**Pull request by buschbapti**
This PR adds the elements in the xacro files to generate a ros2_control compatible URDF.

I also took the liberty to move the generated URDF in a separate folder.

### CODE REVIEW

**Comment by buschbapti** [comment]
@eeberhard

**Review by eeberhard — APPROVED** [review]
I think it's good, hard to tell for sure though! 

It's nice to move generated URDFs to a separate directory and still host them as fixtures for easy usage / testing without needing to generate from scratch. I was thinking this move might break a downstream dependency, but as you mentioned, moving the repository to the aica organisation will have broken any dependency anyway!

File: robots/panda_arm.urdf.xacro [hunk 1]
```diff
@@ -6,12 +6,17 @@
...
-        <xacro:panda_arm arm_id="$(arg arm_id)" connected_to="$(arg connected_to)" xyz="$(arg xyz)" rpy="$(arg rpy)" in_gazebo="true" />
+        <xacro:panda_arm arm_id="$(arg arm_id)" connected_to="$(arg connected_to)" xyz="$(arg xyz)" rpy="$(arg rpy)" in_gazebo="true"/>
     </xacro:if>
 
     <xacro:unless value="$(arg load_gazebo)">  
-        <xacro:panda_arm arm_id="$(arg arm_id)" connected_to="$(arg connected_to)" xyz="$(arg xyz)" rpy="$(arg rpy)" />
+        <xacro:panda_arm arm_id="$(arg arm_id)" connected_to="$(arg connected_to)" xyz="$(arg xyz)" rpy="$(arg rpy)"/>
     </xacro:unless>
+
+    <xacro:panda_arm_interface name="PandaInterface" arm_id="$(arg arm_id)"/>
```
Comment (domire8) [inline robots/panda_arm.urdf.xacro:21]: i know we are not using gazebo but would you need that in both cases you think?

**Review by domire8 — APPROVED** [review]
Thanks

**Pull request merged**